include/mata/alphabet.hh

```diff
@@ -26,6 +26,8 @@
 namespace Mata {
 
 using Symbol = unsigned;
+using Word = std::vector<Symbol>;
```

Comment:
I understand, you probably discussed this with Lukas, but are you sure it is good to introduce new types, after we aggresively removed lots of types? :D

Reply:
We did discuss it. The plan is that `Word` and `WordName` are to be the interface for working with alphabets. And therefore deserve a type of their own. The user will work with word names (a sequence of strings representing names of each symbol in a word), while Mata works internally with words (a sequence of `Mata::Symbol`s). This is the place to discuss this idea with all of us, so feel free to give your opinion.

Reply:
We will keep it, at least until we see how well it works with refactored alphabets (coming soon to you Mata clone!).